apps/ledger-live-mobile/e2e/page/accounts/addAccount.drawer.ts

```diff
@@ -47,4 +60,26 @@ export default class AddAccountDrawer {
     await waitForElementById(this.succesCtaId);
     await tapById(this.succesCtaId);
   }
+
+  async addFirstAccount(currency: Currency) {
+    await this.startAccountsDiscovery();
+    await this.expectAccountDiscovery(currency.name, currency.currencyId);
+    await tapById(this.deselectAllButtonId);
+    await tapById(this.accountCardRegExp(), 0);
+    const accountId = (await getIdOfElement(this.accountCardRegExp(), 0)).replace(
+      /^account-card-/,
+      "",
+    );
+    await this.finishAccountsDiscovery();
+    await this.tapSuccessCta();
+    return accountId;
+  }
+
+  async addAccount(currency: Currency) {
```

Comment:
this method is not in use anywhere, is it expected?

Reply:
Good catch, it is a testing remainder.